### PR TITLE
added login_content template

### PR DESCRIPTION
**Pull request by yceruto**
For more flexibility.

EDIT: The same idea from current templates.

### CODE REVIEW

**Comment by yceruto** [comment]
ping @stof

File: Resources/views/Security/login.html.twig [hunk 1]
```diff
@@ -3,22 +3,5 @@
...
-
-    <label for="password">{{ 'security.login.password'|trans }}</label>
-    <input type="password" id="password" name="_password" required="required" />
-
-    <input type="checkbox" id="remember_me" name="_remember_me" value="on" />
-    <label for="remember_me">{{ 'security.login.remember_me'|trans }}</label>
-
-    <input type="submit" id="_submit" name="_submit" value="{{ 'security.login.submit'|trans }}" />
-</form>
+{% include "FOSUserBundle:Security:login_content.html.twig" %}
```
Comment (xabbuh) [inline Resources/views/Security/login.html.twig]: The `include` tag is deprecated, isn't it?

Comment (javiereguiluz) [inline Resources/views/Security/login.html.twig]: Yes (not technically, but officially). Please always use the `include()` function.

Comment (yceruto) [inline Resources/views/Security/login.html.twig]: Agree, but we shouldn't create a separate PR to change all templates that using this tag?

Comment (yceruto) [inline Resources/views/Security/login.html.twig]: I can create a PR once it has been merged.

**Comment by yceruto** [comment]
Updated!

**Comment by yceruto** [comment]
I should close this PR ?

**Comment by XWB** [comment]
Thanks.

**Pull request merged**